README.md

```diff
@@ -1 +1,9 @@
-# deployment-pipeline
\ No newline at end of file
+# deployment-pipeline
+
+## Scope
+* Define high level pipelines for common cases
+* Architect one CI/Deployment tool for Ops and QA
+* Create working example up to pre-production
+
+### Out of scope
+* Non-trivial changes to existing deployment tooling (cloudformation/ansible/puppet)
```

Comment:
Since we're attempting to work from a blank slate we should look at things and consider their ROI. I created #4 for defining our core values. I think that would help us consider what is in scope or out of scope. I think "non-trivial" is the wrong benchmark.